### PR TITLE
day7 js

### PR DESCRIPTION
**Pull request by jaesung0804**
## What does this PR do?
- Logistic Regression 모델에서 사이즈를 일반화 시켜봤습니다.
- softmax, crossentropyloss, adam 사용했습니다.
- random 생성된 binary 데이터와 iris 데이터로 실험했습니다.
- 결과적으로 두 데이터 모두 문제 없이 loss 감소하고 test accuracy가 1로 좋은 결과를 내는 것을 확인했습니다.

## Does anything else need to be done?
-

### CODE REVIEW

**Review by ok69531 — CHANGES_REQUESTED** [review]
Linear layer를 작성할 때 nn.Linear를 쓰지 않고 작성해주세요
